src/main/java/com/taskadapter/redmineapi/internal/URIConfigurator.java

```diff
@@ -118,41 +115,49 @@ private URI createURI(String query,
         }
     }
 
+    public Collection<? extends NameValuePair> convert(Collection<? extends RequestParam> origParams) {
```

Comment:
this should be a private method. also, "convert" sounds very generic. something like "toNameValue" would be easier to read for people.
plus it should be static